### ISSUE TITLE
can you tell me how to transform "the synth 90k dataset" to the file of tfrecords?

### ISSUE THREAD

**Issue by githubsongsong** [TJCVRS/CRNN_Tensorflow#40]
can you tell me how to transform "the synth 90k dataset" to the file of tfrecords?

thank you sincerely!

**Comment by TJCVRS** [comment]
@githubsongsong I have written a script in tools to write the tfrecords you could refer to it if you want~

**Comment by githubsongsong** [comment]
yes, I see, thank you for your anwser.
 (1) but I want to generate the tfrecords by myself, and I find the ".txt file" of synth 90K' is not normative, the label of every picture is not the word name but the irregular number, so to get my own tfrecords is difficult.    
(2) And I find the content of  "json file" in char_dict of data file is not relevant with the synth 90K, so it makes me confused.

**Comment by TJCVRS** [comment]
@githubsongsong You can use the follows interface to write your own tfrecords.    
with tf.python_io.TFRecordWriter(tfrecords_path) as writer:
            for index, image in enumerate(images):
                features = tf.train.Features(feature={
                    'labels': self.int64_feature(labels[index]),
                    'images': self.bytes_feature(image),
                    'imagenames': self.bytes_feature(imagenames[index])
                })
                example = tf.train.Example(features=features)
                writer.write(example.SerializeToString())

**Comment by githubsongsong** [comment]
![char_dict](https://user-images.githubusercontent.com/31985503/36766225-eadabc1c-1c6f-11e8-8d16-d5b82c008d08.png)

![tfrecords_txt](https://user-images.githubusercontent.com/31985503/36766300-4157094c-1c70-11e8-9928-38bc6ad950ac.png)

**Comment by githubsongsong** [comment]
the above is the content of  "json file" and "synth 90K  .txt file", it is not the same to my think!

**Comment by TJCVRS** [comment]
@githubsongsong You can check the readme.md file to prepare the training data

**Comment by githubsongsong** [comment]
@TJCVRS   ok, thanks you time, I try again.
